### ISSUE TITLE
Centering of text within comments?

### ISSUE THREAD

**Issue by wadehammes** [Stephn-R/atom-section-comments-snippets#1]
The Sublime Text version nicely centers the text for section headers (and doesn't have the unnecessary `>>>>` at the ends), anyway we can get the centering in?

```
/*========================${1/./=/g}
=            ${1:Section comment block}            =
========================${1/./=/g}*/
```

Thats from the Sublime above, I'll try to do a fork later but if there is a reason centering wasn't included by default I'd love to hear why!

Thanks!

**Comment by stephnr** [comment]
Haha I was looking at that EXACTLY when building this plugin but sadly, Atom doesn't support regex in Snippets just yet.

Refer to [Issue #40 on atom-snippets](https://github.com/atom/snippets/issues/40)

**Comment by wadehammes** [comment]
@Stephn-R ah, that makes sense! Well thanks for getting this where it is now, look forward to regex support hopefully!

**Comment by stephnr** [comment]
Agreed! I have subscribed to the topic in atom snippets so we will be
providing a new version of this package as soon as it is released
